### PR TITLE
Remove the use of cookies for `shopOrigin`

### PR DESCRIPTION
**Pull request by hannachen**
### WHY are these changes introduced?
Browsers are slowly dropping 3rd-party cookie support, so partners shouldn't use cookies as the method to store `shopOrigin` when building a new App Bridge app. Instead, developers should be getting the value of `shop` from URL search parameters.

Related to https://github.com/Shopify/app-bridge/issues/1742 https://github.com/Shopify/shopify-dev/pull/4643

### WHAT is this pull request doing?
This PR removes boilerplate code using 3rd-party cookies and cookies-related packages. It also shows partners how to use Next.js to retrieve the `shop` value from the server-side.

Also, clean up some unused imports.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,4 @@
 SHOPIFY_API_KEY="YOUR_SHOPIFY_API_KEY"
 SHOPIFY_API_SECRET="YOUR_SHOPIFY_SECRET"
 HOST="YOUR_TUNNEL_URL"
-SHOP="my-shop-name.myshopify.com"
```
Comment (paulomarg) [inline .env.example:4]: I think this is used by the CLI - does something break if we keep it?

**Review by paulomarg — APPROVED** [review]
I think this all makes sense, save for the one comment!

File: .env.example (hunk 1 above)
Comment (hannachen) [reply .env.example:4]: Nothing will break, I just didn't see any usage within the project. I wasn't aware of the CLI using it, restoring this! 🙂

**Comment by paulomarg** [comment]
🎩 ed this by tweaking the CLI to check this branch out, I was able to create the app, run it and perform OAuth with no problems. LGTM!

**Pull request merged**